src/CMakeLists.txt

```diff
@@ -69,7 +62,7 @@ else()
 endif()
 
 if (${PLATFORM} MATCHES "Web")
-    target_link_options(raylib PRIVATE "-sUSE_GLFW=3")
+    target_link_options(raylib INTERFACE "-sUSE_GLFW=3" "-sASSERTIONS=1")
```

Comment:
again, why was this changed in this PR? I think it should be submitted and reviewed separately.

Reply:
My bad. I guess you're right.